packages/notifications/docs/stories/NotificationMenu.story.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Avatar, Button } from 'react-rainbow-components';
+import { RainbowFirebaseApp } from '@rainbow-modules/app';
+import { NotificationMenu } from '../../src';
+
+const notifications = [
+    {
+        title: 'Rainbow Components',
+        description: 'Lorem ipsum dolor sit amer, adipiscing consectetur',
+        status: 'success',
+        createdAt: 'Now',
+    },
+    {
+        title: 'Rainbow Modules',
+        description: 'Lorem ipsum dolor sit amer, adipiscing consectetur',
+        status: 'success',
+        createdAt: '3 minutes ago',
```

Comment:
here createdAt should be a date or timestamp, also since this will be connected with functions with firebase then we should support firebase timestamps